routes/web.php

```diff
@@ -5,6 +5,7 @@
 Route::prefix('api')->group(function () {
     // Dashboard Routes...
     Route::get('/stats', 'DashboardStatsController@index')->name('horizon.stats.index');
+    Route::post('/clearQueue', 'QueueClearController')->name('horizon.queue-clear.store');
```

Comment:
Unclear to me why a delete operation would be a POST. Shouldn't it be something like DELETE /queues/{queue-name} or something?

Reply:
I thought of clearing queues as more of an action rather than deleting a resource. If you'd like I can change it to a delete operation.